### PR TITLE
Fixes #560: Save reticulate version number used for wrapper generation

### LINKED ISSUE #560: Save the version number for the reticulate package used to generate the wrappers
Save the version number for the reticulate package used to generate the wrappers in the generated wrappers file so it's easy to debug issues as well as making sure the generated wrappers are reproducible.

### PR DESCRIPTION
**Pull request by terrytangyuan**
Fixes #560.

### CODE REVIEW

File: src/interface_r/R/auto_generated_wrappers.R [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # -------------------------------------------------------------------------------
 #             *** WARNING: DO NOT MODIFY THIS FILE ***
 #              
-# Instead, modify scripts/gen_wrappers.R which automatically generates this file
-# ------------------------------------------------------------------------------- 
+# Instead, modify scripts/gen_wrappers.R which automatically generates this file.
+#
+# reticulate version: 1.6.0
```
Comment (navdeep-G) [inline src/interface_r/R/auto_generated_wrappers.R]: I would just changed the wording to: `Generated by reticulate version: ...`

Comment (terrytangyuan) [reply src/interface_r/R/auto_generated_wrappers.R]: Thanks! Changed.

**Comment by navdeep-G** [comment]
LGTM

**Comment by terrytangyuan** [comment]
Merging this since it's just a simple change.

**Pull request merged**